### ISSUE TITLE
About unsupervised set-ups

### ISSUE THREAD

**Issue by Harry-Zhi** [jyhjinghwang/SegSort#1]
Hi,
 Very impressive work on trying unsupervised set-up for semantic segmentation.

Some questions about the codes:
1. Does the unsupervised training still need ImageNet-pretrained weights as an initialisation?
2. Are the results in Table. 3 also obtained with ImageNet-pretrained weights?
3. In Fig.7, in the third "prediction" column. are the black "background" regions obtained by masking directly from Ground Truth or by indirectly voting from background patches?

Thanks.

**Comment by jyhjinghwang** [comment]
Hi Harry,
Thanks for your thoughtful questions.
1) Yes, we do use pre-trained weights for initialization as VOC is too small as a dataset to train on from scratch.  (Even with supervision, the training iterations are too many to achieve satisfactory results.)
2) Yes.  In our opinion, it is a fair comparison as the supervised setting in Table 3 is also initialized the same way.
3) It is by voting from the background patches.
Please let me know if you have further questions.

Jyh-Jing

**Comment by Harry-Zhi** [comment]
Thank you Jyh-Jing a lot for your prompt reply!

**Comment by jyhjinghwang** [comment]
You're welcome! :)
